src/connectedGraph.js

```diff
@@ -0,0 +1,15 @@
+import setCompliment from '../src/setCompliment'
+export default function connectedGraph(graph){
+  let allPoints = Object.keys(graph)
+  let connected = []
+  let toCheck = [allPoints[0]]
+
+  while(toCheck.length>0){
+    let neighbors = graph[toCheck[0]]
+    neighbors.map((points) => toCheck.push(points))
+    connected.push(toCheck.shift())
+    toCheck = setCompliment(connected,toCheck)
+  }
```

Comment:
`I don't immediately understand this logic. Why do you push all the `neighbors` into `toCheck` but then shift off the first one?